.github/workflows/lint.yml

```diff
@@ -38,6 +38,10 @@ jobs:
         run: |
           sudo apt-get install -y doxygen && pip install -r requirements.txt
           cd docs/cpp/source && ./check-doxygen.sh
+      - name: CUDA kernel launch check
+        run: |
+          set -eux
```

Comment:
You sure this works? I feel like a pipefail is necessary as the pipe will suppress exit code from the first python

Reply:
This does correctly identify kernels without protection. I don't know if it will cause a lint failure because it's currently set up to always pass. Once the checks are in place, we can modify the lint check to be binding, so I suppose I'll discover at that point whether or not it integrates with the pipe correctly?

Reply:
This currently doesn't work in any case, since `tee` only uses stdout but `torch/testing/check_kernel_launches.py` only prints to stderr. Could you [replace `|` with `|&`](https://stackoverflow.com/a/363239/5044950)?

Reply:
Fixed.

Reply:
Oh sorry, I was referring to the pipe in the shell command, not the pipe indicating the start of the YAML string 😅

Reply:
Ooops! Fixed that.

Reply:
Thanks! Can you push the fix? I think that's the last thing blocking me from approving